### DISCUSSION TITLE
Ukrainian flag on watermark

### DISCUSSION THREAD

**Discussion by Coospir** [Leaflet/Leaflet#8326] [Q&A] [answered]
## Hello. 
There is a Ukrainian flag on the watermark on the map. Can I remove it? I don't need it. I think it is so stupid to mark it on map. I understand, that Leaflet is free-to-use library, but.. What kind of an imposition is this?

**Comment by Coospir** [comment]
![photo_2022-07-05_13-51-56](https://user-images.githubusercontent.com/22749730/177328772-86afb10e-2ac5-43f9-ab1f-1896ea0159e7.jpg)

**Comment by IvanSanchez** [comment, marked as answer]
See https://github.com/Leaflet/Leaflet/pull/8109 . Please refrain from using the word "stupid".
